blueprints/app/files/.eslintrc.js

```diff
@@ -4,7 +4,8 @@ module.exports = {
     ecmaVersion: 2017,
     sourceType: 'module'
   },
-  extends: 'eslint:recommended',
+  plugins: ['ember'],
+  extends: ['eslint:recommended', 'plugin:ember/recommended'],
```

Comment:
Any plans to also include `ember-best-practices` until it is merged with eslint-plugin-ember?

Reply:
I think having two plugins that do the same thing is quite confusing. We should focus on moving the remaining bits from `ember-best-practices` into the official plugin instead.

Reply:
@Turbo87 - Agreed. I think the plan here needs to be to continue working on bringing over the remaining rules and solidifying things...